### PR TITLE
Create PersonaCard.scss

### PR DESCRIPTION
**Pull request by jenkinsns**

### CODE REVIEW

File: src/components/PersonaCard/PersonaCard.scss [hunk 1]
```diff
@@ -0,0 +1,642 @@
...
+$ms-color-initials-darkRed:     #b91d47;
+
+.ms-PersonaCard
+{
+  color: #333333;
+  font-family: 'Segoe UI Regular WestEuropean', 'Segoe UI', Tahoma, Arial, sans-serif;
+  font-size: 14px;
+  font-weight: normal;
+  animation-name: fadeIn, slideUpIn10;
+    -webkit-animation-duration: 0.167s;
```
Comment (cschlechty) [inline src/components/PersonaCard/PersonaCard.scss]: Fix indentation

File: src/components/PersonaCard/PersonaCard.scss [hunk 2]
```diff
@@ -0,0 +1,642 @@
...
+    background-color: $ms-color-initials-red;
+  }
+  &.ms-Persona-initials--darkRed {
+    background-color: $ms-color-initials-darkRed;
+  }
+}
+
+.ms-Persona-image {
+  display: table-cell;
+  margin-right: 10px;
```
Comment (cschlechty) [inline src/components/PersonaCard/PersonaCard.scss]: Needs localized

File: src/components/PersonaCard/PersonaCard.scss [hunk 3]
```diff
@@ -0,0 +1,642 @@
...
+}
+
+.ms-Persona-presence {
+  background-color: $ms-color-presence-available;
+  position: absolute;
+  height: 12px;
+  width: 12px;
+  border-radius: 50%;
+  top: auto;
+  left: 34px;
```
Comment (cschlechty) [inline src/components/PersonaCard/PersonaCard.scss]: Needs localized

File: src/components/PersonaCard/PersonaCard.scss [hunk 4]
```diff
@@ -0,0 +1,642 @@
...
+    overflow: visible;
+    background: transparent;
+    height: 0;
+    width: 0;
+  }
+
+  .ms-Persona-presence {
+    right: auto;
+    top: 10px;
+    left: 0;
```
Comment (cschlechty) [inline src/components/PersonaCard/PersonaCard.scss]: localize these RTL values

File: src/components/PersonaCard/PersonaCard.scss [hunk 5]
```diff
@@ -0,0 +1,642 @@
...
+      border: 1px solid $ms-color-white;
+    }
+
+    @media screen and (-ms-high-contrast: black-on-white) {
+      border: 1px solid $ms-color-black;
+    }
+  }
+
+  .ms-Persona-details {
+    padding-left: 20px;
```
Comment (cschlechty) [inline src/components/PersonaCard/PersonaCard.scss]: localize

File: src/components/PersonaCard/PersonaCard.scss [hunk 6]
```diff
@@ -0,0 +1,642 @@
...
+    top: 6px;
+  }
+
+  .ms-Persona-initials {
+    font-size: $ms-font-size-s;
+    line-height: 32px;
+  }
+
+  .ms-Persona-presence {
+    left: 19px;
```
Comment (cschlechty) [inline src/components/PersonaCard/PersonaCard.scss]: localize

File: src/components/PersonaCard/PersonaCard.scss [hunk 7]
```diff
@@ -0,0 +1,642 @@
...
+  }
+
+  .ms-Persona-details {
+    padding-left: 8px;
```
Comment (cschlechty) [inline src/components/PersonaCard/PersonaCard.scss]: localize

File: src/components/PersonaCard/PersonaCard.scss [hunk 8]
```diff
@@ -0,0 +1,642 @@
...
+}
+
+
+//== Modifier: Extra Small Square Persona
+//
+.ms-Persona.ms-Persona--square.ms-Persona--xs {
+  .ms-Persona-presence {
+    height: 32px;
+    width: 4px;
+    left: 0;
```
Comment (cschlechty) [inline src/components/PersonaCard/PersonaCard.scss]: localize

File: src/components/PersonaCard/PersonaCard.scss [hunk 9]
```diff
@@ -0,0 +1,642 @@
...
+    top: 5px;
+  }
+
+  .ms-Persona-initials {
+    font-size: $ms-font-size-m;
+    line-height: 40px;
+  }
+
+  .ms-Persona-presence {
+    left: 27px;
```
Comment (cschlechty) [inline src/components/PersonaCard/PersonaCard.scss]: localize

File: src/components/PersonaCard/PersonaCard.scss [hunk 10]
```diff
@@ -0,0 +1,642 @@
...
+  }
+
+  .ms-Persona-details {
+    padding-left: 8px;
```
Comment (cschlechty) [inline src/components/PersonaCard/PersonaCard.scss]: localize

File: src/components/PersonaCard/PersonaCard.scss [hunk 11]
```diff
@@ -0,0 +1,642 @@
...
+}
+
+
+//== Modifier: Small Square Persona
+//
+.ms-Persona.ms-Persona--square.ms-Persona--sm {
+  .ms-Persona-presence {
+    height: 40px;
+    width: 4px;
+    left: 0;
```
Comment (cschlechty) [inline src/components/PersonaCard/PersonaCard.scss]: localize

File: src/components/PersonaCard/PersonaCard.scss [hunk 12]
```diff
@@ -0,0 +1,642 @@
...
+}
+
+
+//== Modifier: Large Square Persona
+//
+.ms-Persona.ms-Persona--square.ms-Persona--lg {
+  .ms-Persona-presence {
+    height: 72px;
+    width: 7px;
+    left: 0;
```
Comment (cschlechty) [inline src/components/PersonaCard/PersonaCard.scss]: loc

File: src/components/PersonaCard/PersonaCard.scss [hunk 13]
```diff
@@ -0,0 +1,642 @@
...
+
+  .ms-Persona-initials {
+    font-size: $ms-font-size-su;
+    line-height: 100px;
+  }
+
+  .ms-Persona-presence {
+    height: 20px;
+    width: 20px;
+    left: 71px;
```
Comment (cschlechty) [inline src/components/PersonaCard/PersonaCard.scss]: loc

File: src/components/PersonaCard/PersonaCard.scss [hunk 14]
```diff
@@ -0,0 +1,642 @@
...
+  }
+
+  .ms-Persona-details {
+    padding-left: 20px;
```
Comment (cschlechty) [inline src/components/PersonaCard/PersonaCard.scss]: loc

File: src/components/PersonaCard/PersonaCard.scss [hunk 15]
```diff
@@ -0,0 +1,642 @@
...
+}
+
+
+//== Modifier: Extra Large Square Persona
+//
+.ms-Persona.ms-Persona--square.ms-Persona--xl {
+  .ms-Persona-presence {
+    height: 100px;
+    width: 9px;
+    left: 0;
```
Comment (cschlechty) [inline src/components/PersonaCard/PersonaCard.scss]: loc

File: src/components/PersonaCard/PersonaCard.scss [hunk 16]
```diff
@@ -0,0 +1,642 @@
...
+    background-color: $ms-color-presence-offline;
+  }
+}
+
+// TODO: Remove overrides below
+
+// div.ms-Persona-image to overwrite Fabric's ms-Persona-image
+div.ms-Persona-image {
+  position: absolute;
+  left: 0;
```
Comment (cschlechty) [inline src/components/PersonaCard/PersonaCard.scss]: loc

**Comment by ericthompson** [comment]
Hi @jenkinsns - we're currently going through some major design work on the PersonaCard that should land in the near future. In addition to being rebuilt, it will also involve significant design changes. This means that we won't be able to take a PersonaCard submission for the toolkit at the moment, but I have gone ahead and added this to the [Trello board](https://trello.com/c/OOCSXyS3/10-persona-person-card) (along with my vote for its inclusion!).

**Pull request closed without merging**